typescript/api-cors-lambda-crud-dynamodb/index.ts

```diff
@@ -14,62 +16,94 @@ export class ApiLambdaCrudDynamoDBStack extends cdk.Stack {
+       * the new table, and it will remain in your account until manually deleted. By setting the policy to
+       * DESTROY, cdk destroy will delete the table (even if it has data in it)
+       */
       removalPolicy: cdk.RemovalPolicy.DESTROY, // NOT recommended for production code
     });
 
-    const getOneLambda = new lambda.Function(this, 'getOneItemFunction', {
-      code: new lambda.AssetCode('src'),
-      handler: 'get-one.handler',
-      runtime: lambda.Runtime.NODEJS_10_X,
+    const getOneLambda = new NodejsFunction(this, 'getOneItemFunction', {
+      entry: join(__dirname, 'lambdas', 'get-one.ts'), // accepts .js, .jsx, .ts and .tsx files
+      depsLockFilePath: join(__dirname, 'lambdas', 'package-lock.json'),
+      handler: 'handler',
+      runtime: lambda.Runtime.NODEJS_14_X,
+      bundling: {
+        externalModules: [
+          'aws-sdk', // Use the 'aws-sdk' available in the Lambda runtime
+        ],
+      },
       environment: {
         TABLE_NAME: dynamoTable.tableName,
         PRIMARY_KEY: 'itemId'
       }
     });
 
-    const getAllLambda = new lambda.Function(this, 'getAllItemsFunction', {
-      code: new lambda.AssetCode('src'),
-      handler: 'get-all.handler',
-      runtime: lambda.Runtime.NODEJS_10_X,
+    const getAllLambda = new NodejsFunction(this, 'getAllItemsFunction', {
+      entry: join(__dirname, 'lambdas', 'get-all.ts'), // accepts .js, .jsx, .ts and .tsx files
+      depsLockFilePath: join(__dirname, 'lambdas', 'package-lock.json'),
+      handler: 'handler',
+      runtime: lambda.Runtime.NODEJS_14_X,
+      bundling: {
+        externalModules: [
+          'aws-sdk', // Use the 'aws-sdk' available in the Lambda runtime
+        ],
+      },
       environment: {
         TABLE_NAME: dynamoTable.tableName,
         PRIMARY_KEY: 'itemId'
       }
     });
 
-    const createOne = new lambda.Function(this, 'createItemFunction', {
-      code: new lambda.AssetCode('src'),
-      handler: 'create.handler',
-      runtime: lambda.Runtime.NODEJS_10_X,
+    const createOne = new NodejsFunction(this, 'createItemFunction', {
+      entry: join(__dirname, 'lambdas', 'create.ts'), // accepts .js, .jsx, .ts and .tsx files
+      depsLockFilePath: join(__dirname, 'lambdas', 'package-lock.json'),
+      handler: 'handler',
+      runtime: lambda.Runtime.NODEJS_14_X,
+      bundling: {
+        externalModules: [
+          'aws-sdk', // Use the 'aws-sdk' available in the Lambda runtime
+        ],
+      },
       environment: {
         TABLE_NAME: dynamoTable.tableName,
         PRIMARY_KEY: 'itemId'
       }
     });
 
-    const updateOne = new lambda.Function(this, 'updateItemFunction', {
-      code: new lambda.AssetCode('src'),
-      handler: 'update-one.handler',
-      runtime: lambda.Runtime.NODEJS_10_X,
+    const updateOne = new NodejsFunction(this, 'updateItemFunction', {
+      entry: join(__dirname, 'lambdas', 'update-one.ts'), // accepts .js, .jsx, .ts and .tsx files
```

Comment:
Imo these comments only need to appear on the first occurrence (unless you have strong feelings to the contrary).